packages/wonder-blocks-clickable/components/clickable.js

```diff
@@ -77,6 +77,7 @@ export default class Clickable extends React.Component<Props> {
                 <StyledLink
                     {...commonProps}
                     to={this.props.href}
+                    onClick={this.props.onClick}
```

Comment:
Any particular reason for moving `onClick` here. It seems that was working before as part of `commonProps`, right?

Reply:
sorry I'll merge from the Clickable branch here, it's been addressed there, this is outdated